Gemfile

```diff
@@ -24,18 +26,15 @@ gem "openssl"
 
 gem "deface"
 
-# if deploying to a dedicated server
-# gem "daemons"
-# gem "delayed_job_active_record"
-# gem "whenever"
```

Comment:
we must uncomment that and comment heroku deploying gems.

Reply:
Look at the complete file, all that lines are uncommented but was duplicated so that's why I removed them